docs/users-guide/saved-questions.md

```diff
@@ -0,0 +1,91 @@
+
+We could create this question using the notebook editor, like so:
+
+![Example notebook](images/saved-questions/example-notebook.png).
+
+Alternatively, we could create that question using SQL:
+
+```
+SELECT *
+FROM   orders AS o
+       INNER JOIN products AS p
+               ON o.product_id = p.id
+WHERE  p.category = 'Gizmo'
+       AND o.created_at BETWEEN '2019-01-01' AND '2019-12-31'
+```
+
+We'll save that question as "Gizmo orders in 2019".
+
+Now let's refer to that "Gizmo orders in 2019" in a new query. To keep it simple, let's say we just want to count all of the Gizmo orders from 2019. We can use the `#` symbol to refer to a saved question in a query.
+
+If we type out:
+
+```
+SELECT count(*)
+FROM {% raw %}{{#{% endraw %}
+```
+
+Metabase will slide out a sidebar where we can select a question to reference. We'll search for our "Gizmo orders in 2019" question
+
+![Select a question from the variable sidebar](images/saved-questions/variable-sidebar.png)
+
+We'll select our "Gizmo orders in 2019" question, and Metabase will update our code with the question's ID, `5`:
+
+```
+SELECT count(*)
+FROM {% raw %}{{#5}}{% endraw %}
+```
+
+This query returns the number of records from our saved question, "Gizmo orders in 2019", which has an ID of `5`.
+
+## Saved question as a Common Table Expression (CTE)
+
+The same syntax can be used in [Common Table Expressions (CTEs)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) (with SQL databases that support CTEs):
+
+```
+WITH 2019_gizmo_orders AS {% raw %}{{#5}}{% endraw %}
+SELECT count(*)
+FROM 2019_gizmo_orders
+```
+
+The `{% raw %}{{#5}}{% endraw %}` tag is substituted for the SQL query of the referenced question, surrounded by parentheses, equivalent to:
+
+```
+WITH 2019_gizmo_orders AS (SELECT *
+FROM   orders AS o
+       INNER JOIN products AS p
+               ON o.product_id = p.id
+WHERE  p.category = 'Gizmo'
+       AND o.created_at BETWEEN '2019-01-01' AND '2019-12-31')
+SELECT count(*)
+FROM 2019_gizmo_orders
+```
+
+# How to find a question's ID
+
+- Selecting a question from the variable sidebar in the SQL editor will automatically add the ID number to the variable in our query.
+- We can also load the question we'd like to reference and find the ID at the end of the URL in your browser's address bar, after `/question/`, e.g., for `https://our.metabase.com/question/12345`, the question's ID would be `12345`.
+
+## Use cases for saved questions
+
+Why you would use a saved question as a data source:
+
+- You can't create a view in the database, as saved questions effectively act as views. If you can create a view, and you expect that you and others will frequently query the results, consider creating a materialized view. The results will be stored in the database (as opposed to computed each time), which will speed up query time.
```

Comment:
```suggestion
- If you can't create a view in the database, since saved questions effectively act as views. If you can create a view, and you expect that you and others will frequently query the results, consider creating a materialized view. The results will be stored in the database (as opposed to computed each time), which will speed up query time.
```